addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -316,6 +316,9 @@ var SnippetEditor = Widget.extend({
         });
         this._customize$Elements.forEach(($el, i) => {
             var editor = $el.data('editor');
+            if (!editor) {
+                return;
+            }
```

Comment:
Because the theme customization widget's element was inserted into the _customize$Elements array and it has no editor data associated with it. If you're not getting an error you probably separated those concerns and it's probably no longer needed.

Reply:
I did not change anything and does not seem needed anymore ;)